### PR TITLE
Integrated an initial changelog

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  * Refactored Docker container [#104](https://github.com/go-gitea/gitea/issues/104)
+  * Added short-hash support for downloads [#211](https://github.com/go-gitea/gitea/issues/211)
+  * Display tooltip for downloads [#221](https://github.com/go-gitea/gitea/issues/221)
+  * Improved HTTP headers for issue attachments [#270](https://github.com/go-gitea/gitea/pull/270)
+  * Integrate public as bindata optionally [#293](https://github.com/go-gitea/gitea/pull/293)
+  * Integrate templates as bindata optionally [#314](https://github.com/go-gitea/gitea/pull/314)
+  * Inject more ENV variables into custom hooks [#316](https://github.com/go-gitea/gitea/issues/316)
+  * Correct LDAP login validation [#342](https://github.com/go-gitea/gitea/pull/342)
+  * Integrate conf as bindata optionally [#354](https://github.com/go-gitea/gitea/pull/354)
+  * Serve vide files in browser [#418](https://github.com/go-gitea/gitea/pull/418)
```
Comment (lunny) [inline CHANGELOG.md]: vide - > video

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+* BUGFIXES
+  * Fixed sender of notifications [#2](https://github.com/go-gitea/gitea/pull/2)
+  * Fixed keyword hijacking vulnerability [#20](https://github.com/go-gitea/gitea/pull/20)
+  * Fixed non-markdown readme rendering [#95](https://github.com/go-gitea/gitea/pull/95)
+  * Allow updating draft releases [#169](https://github.com/go-gitea/gitea/pull/169)
+  * GitHub API compliance [#227](https://github.com/go-gitea/gitea/pull/227)
+  * Added commit SHA to tag webhook [#286](https://github.com/go-gitea/gitea/issues/286)
+  * Secured links via noopener [#315](https://github.com/go-gitea/gitea/issues/315)
+  * Replace tabs with spaces on wiki title [#371](https://github.com/go-gitea/gitea/pull/371)
+  * Fixed vulnerability on labels [#409](https://github.com/go-gitea/gitea/pull/409)
```
Comment (lunny) [inline CHANGELOG.md]: And releases

**Review by lunny — CHANGES_REQUESTED** [review]

**Comment by lunny** [comment]
Maybe 1.0.txt is better than changelog

**Comment by tboerger** [comment]
> Maybe 1.0.txt is better than changelog

Oh no, don't do that. The CHANGELOG.md is pretty conventional and it gets updated with every release. Later on we will add some scripting that generates it automatically like Gitlab.

File: CHANGELOG.md (hunk 2 above)
Comment (tboerger) [reply CHANGELOG.md]: Fixed

File: CHANGELOG.md (hunk 1 above)
Comment (tboerger) [reply CHANGELOG.md]: Fixed

**Review by lunny — APPROVED** [review]

**Comment by lunny** [comment]
LGTM

**Comment by Bwko** [comment]
LGTM

**Pull request merged**